Source/Core/DolphinQt/Settings/WiiPane.cpp

```diff
@@ -253,6 +263,16 @@ void WiiPane::OnSaveConfig()
   Config::SetBase(Config::MAIN_ALLOW_SD_WRITES, m_allow_sd_writes_checkbox->isChecked());
   Settings::Instance().SetUSBKeyboardConnected(m_connect_keyboard_checkbox->isChecked());
 
+  std::string nus_url = m_nus_url_edit->text().toStdString();
```

Comment:
Since `text()` returns a QString, wouldn't it be a little nicer to use Qt's built in QString members to do the compare and insert below? It'd get rid of the conversion to/from std::string